p2p/host/autonat/autonat.go

```diff
@@ -399,33 +399,30 @@ func (as *AmbientAutoNAT) getPeerToProbe() peer.ID {
 		return ""
 	}
 
-	candidates := make([]peer.ID, 0, len(peers))
+	// clean old probes
```

Comment:
~Do we need to decrement pending probes?~

Nevermind, this is about our throttling, not about outstanding requests. May help to wordsmith the comment.